nerfstudio/data/datamanagers/base_datamanager.py

```diff
@@ -313,9 +317,9 @@ class VanillaDataManagerConfig(DataManagerConfig):
     """Specifies the dataparser used to unpack the data."""
     train_num_rays_per_batch: int = 1024
     """Number of rays per batch to use per training iteration."""
-    train_num_images_to_sample_from: int = -1
```

Comment:
this is just for tesing / hacking, or are you proposing these params could be good for everybody?  Either way if you happen to decide on new parameters you think are good for everybody, good to highlight and discuss that in the summary and/or discussion portion of the Pull Request

Reply:
Yep this is just some hacking for now - I have not done too many tests changing these figures around